src/settings/components/utils.ts

```diff
@@ -572,14 +572,19 @@ export function calculateNewIdentitiesState(
 /**
  * Format a size in byte as human-readable
  */
-export const humanFileSize = (inputSize: number): string => {
+export const humanFileSize = (inputSize: number, t: TFunction | undefined): string => {
+	const units = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
 	if (inputSize === 0) {
-		return '0 B';
+		const unit = units[0];
+		const translatedUnit = t ? t('size.unitMeasure', { context: unit, defaultValue: unit }) : unit;
+		return `0 ${translatedUnit}`;
```

Comment:
can we deduplicate?
maybe something like const i = inputSize === 0 ? 0 : Math.floor(Math.log(inputSize) / Math.log(1024));